.github/workflows/ci.yml

```diff
@@ -223,3 +225,44 @@ jobs:
         with:
           path: perf-measures/type-check/previous-result.txt
           key: type-check-perf-previous-result-${{ github.sha }}
+
+  perf-measures-bundle-check-on-pr:
+    name: 'Bundle Check on PR'
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: actions/checkout@v4
+      - uses: oven-sh/setup-bun@v2
+      - run: bun install
+      - run: bun run build
+      - run: bunx esbuild --minify --bundle dist/index.js --format=esm --outfile=perf-measures/bundle-check/generated/after.js
+      - uses: actions/cache/restore@v4
+        with:
+          path: perf-measures/bundle-check/generated/before.js
+          restore-keys: |
+            perf-measures-bundle-check-previous-file-
+          key: perf-measures-bundle-check-previous-file-
+      - run: |
+          {
+            echo 'COMPARISON<<EOF'
+            ls -l ./generated | column -s '|' -t
```

Comment:
It seems that it is not enough to simply parse the result of the ls command to get output like the following
https://github.com/honojs/hono/actions/runs/11818332808/job/32925651835?pr=3665#step:9:24

The `column -s '|' -t` command formats `|` delimited characters, so you need to pipe at least `|` delimited output

ex)
```
| | Current | Previous |
| --- | --- | --- |
| Files | 256 | 256 |
| Lines | 115024 | 115024 |
| Identifiers | 112049 | 112049 |
| Symbols | 243346 | 243346 |
| Types | 204684 | 204684 |
| Instantiations | 3044052 | 3044052 |
| Memory used | 422387K | 422692K |
| I/O read | 0.01s | 0.02s |
| I/O write | 0.00s | 0.00s |
| Parse time | 0.28s | 0.28s |
| Bind time | 0.11s | 0.11s |
| Check time | 2.40s | 2.42s |
| Emit time | 0.00s | 0.00s |
| Total time | 2.79s | 2.81s |
```

Reply:
thanks, any idea to format the output?

Reply:
thanks, any idea to format the output?

Reply:
How about the following format?

| | Current | Previous |
| --- | --- | --- |
|  Bundle size | 19206 | 19206 |

The number of bytes assumes that the number of digits will not change in the future.

Reply:
Looks good to me!

Reply:
![image](https://github.com/user-attachments/assets/1a55d5a2-60ff-4577-ac5e-dac36bba6320)
@k2tzumi what do you think?

Reply:
It is simple, but it is good enough for now.
Let's add custom metrics later.